### PR TITLE
fix #9172 chore(project): add cirrus and schemas to dependabot

### LINKED ISSUE #9172: Add cirrus and schemas to dependabot
We now have some additional projects living in this repo, for cirrus and nimbus schemas.  Let's add them to our dependabot config so they can also receive automated package updates.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3714)

### PR DESCRIPTION
**Pull request by jaredlockhart**
Because

* We now have cirrus and schemas projects living in this repo
* Everyone benefits from automated package updates

This commit

* Adds cirrus and schemas to the dependabot config

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ updates:
     ignore:
       - dependency-name: "*"
         update-types:
-          ["version-update:semver-major", "version-update:semver-patch"]
+          ["version-update:semver-major", "version-update:semver-minor", "version-update:semver-patch"]
```
Comment (mikewilli) [inline .github/dependabot.yml:11]: I'm confused -- won't this ignore all version updates? (and same question for the below `ignore.update-types` sections)

File: .github/dependabot.yml [hunk 2]
```diff
@@ -26,4 +26,22 @@ updates:
...
+  - package-ecosystem: pip
+    directory: "/cirrus/server"
+    schedule:
+      interval: weekly
+    target-branch: main
+    ignore:
+      - dependency-name: "*"
+        update-types:
+          ["version-update:semver-major", "version-update:semver-minor", "version-update:semver-patch"]
+  - package-ecosystem: pip
```
Comment (mikewilli) [inline .github/dependabot.yml:39]: I'll have to add `npm` to this config in the PR that adds npm publishing to schemas.

**Review by yashikakhurana — APPROVED** [review]
thank you @jaredlockhart that totally makes sense

**Review by mikewilli — CHANGES_REQUESTED** [review]
I might just be confused about how the `ignore.update-types` works, but can you please clarify dependabot's behavior under this configuration?

File: .github/dependabot.yml (hunk 1 above)
Comment (jaredlockhart) [reply .github/dependabot.yml:11]: Oh duh!  You're right it's ignore, not include XD  I'll fix that.

File: .github/dependabot.yml (hunk 2 above)
Comment (jaredlockhart) [reply .github/dependabot.yml:39]: Yep 👍

**Pull request merged**